### PR TITLE
[New Entry] SQL Commands: `SELECT TOP`

### LINKED ISSUE #1303: [Entry] SQL: SELECT TOP
### Reason/inspiration (optional)

There are entries for similar SQL commands such as LIMIT but I noticed that this did not have one yet.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by jmpos7**
### Description

Adding a basic new entry for the TOP command, which did not have an entry. Adding to the Codecademy documentation is a project in the Learn Git & GitHub course and I wanted to make a contribution.

Closes #1303

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by SSwiniarski** [comment]
@jmpos7, thanks for your contribution. Please remember to sign the [CLA](https://cla-assistant.io/Codecademy/docs?pullRequest=1305).

**Comment by Dusch4593** [comment]
Hi @jmpos7, wanted to also confirm that this pull request aims to fix issue #1303?

**Comment by Dusch4593** [comment]
@jmpos7 awesome thanks! And double-thanks for signing the CLA 👍🏻

File: content/sql/concepts/commands/terms/top/top.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+Returns a specified number of rows from the top of the result.
+
+## Syntax
+
+```sql
```
Comment (SSwiniarski) [inline content/sql/concepts/commands/terms/top/top.md:20]: The syntax section needs a few changes:
- The examples should be in a ```pseudo block.
- The examples should be generalized, such as `SELECT TOP (n)` `SELECT TOP (n) PERCENT`
- Each example should have an explanatory sentence describing what it's doing and what `n` is.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@jmpos7 , I did a first review and left one comment to address.

**Comment by Dusch4593** [comment]
Hey @jmpos7! 👋🏻 Did you have any questions or need any assistance with implementing the feedback?

**Comment by jmpos7** [comment]
@Dusch4593 Thanks for checking in -- I made the reviewer recommended changes to the document. What is the next step?

File: content/sql/concepts/commands/terms/top/top.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+Returns a specified number of rows from the top of the result.
+
+## Syntax
+
+This command is used to select the inital rows from a table, limiting the result to a specified number, represented here by (n):
```
Comment (SSwiniarski) [inline content/sql/concepts/commands/terms/top/top.md:20]:
```suggestion
This command is used to select the inital rows from a table, limiting the result to a specified number, represented here by `n`:
```

File: content/sql/concepts/commands/terms/top/top.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+```sql
+SELECT TOP (n) column_name(s)
+FROM table_name;
+```
+
+The command can also be used with PERCENT to limit the result to the top (n) percent of rows:
```
Comment (SSwiniarski) [inline content/sql/concepts/commands/terms/top/top.md:27]:
```suggestion
The command can also be used with PERCENT to limit the result to the top `n` percent of rows:
```

**Review by SSwiniarski — APPROVED** [review]
@jmpos7 I had two more minor changes, but otherwise looks good for a second review.

File: content/sql/concepts/commands/terms/top/top.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+---
+Title: 'TOP'
```
Comment (Dusch4593) [inline content/sql/concepts/commands/terms/top/top.md:2]: I'm wondering if this entry should be repurposed as a concept entry and renamed as 'SELECT TOP' (similar to the existing [SELECT DISTINCT](https://www.codecademy.com/resources/docs/sql/commands/select-distinct) entry).

cc: @SSwiniarski

File: content/sql/concepts/commands/terms/top/top.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+Tags:
+  - 'Database'
+  - 'Queries'
+  - 'SQL Server'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+---
+
+Returns a specified number of rows from the top of the result.
```
Comment (Dusch4593) [inline content/sql/concepts/commands/terms/top/top.md:16]:
```suggestion
The **`TOP`** command returns a specified number of rows from the top of the result.
```

The opening line of the entry should begin with the concept/term at hand, and the first use of it should be **bolded** as suggested above.

File: content/sql/concepts/commands/terms/top/top.md (hunk 3 above)
Comment (Dusch4593) [reply content/sql/concepts/commands/terms/top/top.md:27]: To add to this suggestion, let's surround "PERCENT" in backticks (i.e., \`PERCENT\`)

File: content/sql/concepts/commands/terms/top/top.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+This command is used to select the inital rows from a table, limiting the result to a specified number, represented here by (n):
+
+```sql
+SELECT TOP (n) column_name(s)
+FROM table_name;
+```
+
+The command can also be used with PERCENT to limit the result to the top (n) percent of rows:
+
+```sql
```
Comment (Dusch4593) [inline content/sql/concepts/commands/terms/top/top.md:29]:
```suggestion
```pseudo
```

File: content/sql/concepts/commands/terms/top/top.md (hunk 1 above)
Comment (Dusch4593) [reply content/sql/concepts/commands/terms/top/top.md:22]: I would also include a sentence in the "Syntax" section that explains how `column_name(s)` is a space-separated list of existing columns that are selected from `table_name`.

File: content/sql/concepts/commands/terms/top/top.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+SELECT TOP (n) PERCENT column_name(s)
+FROM table_name;
+```
+
+## Examples
+
+The given query will display the top 5 rows from the `books` table.
```
Comment (Dusch4593) [inline content/sql/concepts/commands/terms/top/top.md:36]:
```suggestion
The following query selects all columns from the top 5 rows of the `books` table:
```

File: content/sql/concepts/commands/terms/top/top.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+```sql
+SELECT TOP (5) *
+FROM books;
+```
+
+The given query will display the top 25% of rows from the `films` table.
```
Comment (Dusch4593) [inline content/sql/concepts/commands/terms/top/top.md:43]:
```suggestion
The next query selects the top 25% of rows from the `movies` table, under just the `movie_titles` column:
```

**Review by Dusch4593 — DISMISSED** [review]
@jmpos7 Left you some comments/suggestions but I may have more after this current feedback is implemented. 😄

**Comment by SSwiniarski** [comment]
Congrats @jmpos7! Your new entry is live here: https://www.codecademy.com/resources/docs/sql/commands/select-top

It looks like you're currently anonymous.
image
If you'd like to become public, make sure to add your GitHub username to your Codecademy profile and set your visibility to everyone.

**Pull request merged**